tests/Parsers/PHPReflectionParser.php

```diff
@@ -15,40 +15,44 @@ class PHPReflectionParser
 {
     public static function getStubs(): StubsContainer
     {
-        $stubs = new StubsContainer();
-
-        $jsonData = json_decode(file_get_contents(__DIR__ . '/../TestData/mutedProblems.json'));
-        $const_groups = get_defined_constants(true);
-        unset($const_groups['user']);
-        $const_groups = Utils::flattenArray($const_groups, true);
-        foreach ($const_groups as $name => $value) {
-            $constant = (new PHPDefineConstant())->readObjectFromReflection([$name, $value]);
-            $constant->readMutedProblems($jsonData->constants);
-            $stubs->addConstant($constant);
-        }
+        if (file_exists(__DIR__ . '/../../ReflectionData.json')) {
```

Comment:
If the file exists (for example, locally), it won't be regenerated on serialization. We can either show a message or better yet avoid this possible confusion by passing the argument that we need to reserealize.

Reply:
This check is done intentionally to avoid generating of `ReflectionData.json` locally if one just want to run tests with some PHP version and don't want to generate file to check other PHP versions. (some kind of fallback behavior). In case of a run on CI  `ReflectionData.json` will be always generated. I think that for local development it's easier than adding some additional argument - just run it as before.